client/my-sites/domains/domain-management/transfer/transfer-page/index.tsx

```diff
@@ -75,8 +83,17 @@ const TransferPage = ( props: TransferPageProps ): JSX.Element => {
 					headerText={ __( 'To another user' ) }
 					mainText={ __( 'Transfer this domain to any administrator on this site' ) }
 				/>
-				<div className="transfer-page__item-separator"></div>
+			);
+		}
+
+		if ( ! ( isPrimaryDomain && isAtomic ) ) {
```

Comment:
The condition in the original `domain-management/transfer/index.jsx` file is `( isAtomic && ! isPrimaryDomain ) || ! isAtomic )`, which is equivalent to this.